.schema/api.openapi.json

```diff
@@ -1932,6 +1932,81 @@
         ]
       }
     },
+    "/self-service/saml/metadata":{
+      "get":{
+        "description": "This endpoint is for the IDP to obtain kratos metadata",
```

Comment:
The style for the other descriptions is slightly different.

"This endpoints returns kratos metadata as part of a SAMl login" would be closer to what is done on other endpoints.